.github/workflows/release.yml

```diff
@@ -54,4 +54,7 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       run: |
-        gh release create $ORT_SERVER_VERSION --notes-file RELEASE_NOTES.md
+        if [[ "$ORT_SERVER_VERSION" =~ "-RC[0-9]+" ]]; then
```

Comment:
In my tests, the regex must not be quoted to match, so omit `"` around `-RC[0-9]+`.